roles/stressng/templates/stressng_workload.yml.j2

```diff
@@ -60,8 +73,6 @@ spec:
             value: "{{ elasticsearch.index_name | default("ripsaw-stressng") }}"
           - name: parallel
             value: "{{ elasticsearch.parallel | default(false) }}"
-          - name: es_verify_cert
-            value: "{{ elasticsearch.verify_cert | default(true) }}"
```

Comment:
We should retain this capability for use with self-signed certs.

Reply:
Thx, Fix it.